update_repos.sh

```diff
@@ -1,16 +1,17 @@
 #! /usr/bin/env bash
 folder="./workspace-zen"
+docker-compose pull
 declare -a repos=("cp-badges-service"
-                  "cp-dojos-service"
-                  "cp-eventbrite-service"
-                  "cp-events-service"
-                  "cp-organisations-service"
-                  "cp-translations"
-                  "cp-users-service"
-                  "cp-zen-frontend"
-                  "cp-zen-platform")
+  "cp-dojos-service"
+  "cp-eventbrite-service"
+  "cp-events-service"
+  "cp-organisations-service"
+  "cp-translations"
+  "cp-users-service"
+  "cp-zen-frontend"
+  "cp-zen-platform")
 for repo in "${repos[@]}"; do
   cd $folder/"$repo" || (echo "Couldn't access $folder/$repo" && exit)
-  git pull
+  git fetch origin
```

Comment:
fetch doesn't update the reset the current branch, it's fairly different from pull

Reply:
yes but if you do a pull in a forloop you could end up with a lot of merge conflicts